data-raw/combine_and_clean_data.R

```diff
@@ -0,0 +1,82 @@
+library(tidyverse)
+
+source("data-raw/early_snorkel_db_pull.R")
+source("data-raw/revised_snorkel_db_pull.R")
+
+
+# look at data early
+glimpse(cleaner_snorkel_data_early)
+glimpse(cleaner_snorkel_metadata_early)
+
+
+# look at data revised
+glimpse(cleaner_snorkel_observations)
+glimpse(cleaner_snorkel_survey_metadata)
+
+
+# snorkel obs data
+sort(colnames(cleaner_snorkel_data_early))
+sort(colnames(cleaner_snorkel_observations))
+# columns: observation_id, survey_id, unit, count, fork_length, substrate, instream_cover, overhead_cover, hydrology, depth, species, clipped
+# TODO column discrepancies - velocity only in early data
+
+# snorkel survey metadata data
+# columns: survey_id, date, flow, section_name, section_number, survey_id, temperature, turbidity, units_covered, weather
+# TODO: survey_names are still a mess in the early data
+sort(colnames(cleaner_snorkel_metadata_early))
+sort(colnames(cleaner_snorkel_survey_metadata))
+
+# unit lookup - table to join sampling unit to river mile/section/section type/channel
+sampling_unit_lookup |> glimpse()
+
+# combine and clean
+# FEATHER SNORKEL OBS ----------------------------------------------------------
+combined_snorkel_observations <- bind_rows(cleaner_snorkel_data_early |> mutate(instream_cover = as.character(instream_cover)),
+                                           cleaner_snorkel_observations) |>
+  filter(!unit %in% c("77-80", "86-89",
+                      "104, 106, 112", "104 106  112",
+                      "104 106 112", "446/449"), # filtered out these messy units for now, alternatively we can see if casey can assign a non messy unit
+         species != "Sacramento Squawfish Or Hardhead") |> glimpse()
```

Comment:
Lets just change name to sacramento pikeminnow

Reply:
updated